src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -91,7 +92,8 @@
     /**
      * The map of bridge JID to <tt>BridgeState</tt>.
      */
-    private final Map<String, BridgeState> bridges = new HashMap<>();
+    private final Map<String, BridgeState> bridges
+        = new ConcurrentHashMap<>();
```

Comment:
Replacing this with making getKnownBridgesCount() synchronized (I now see this is only used in tests).